styles.css

```
@@ -8136,6 +8136,96 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+    background: var(--highlight);
+    border-color: var(--highlight);
+    box-shadow: 0 0 0 1px rgb(var(--highlight-rgb), 0.3);
+}
+
+.eq-spectrum-pill.active .pill-value,
+.eq-spectrum-pill.active .pill-label {
+    color: inherit;
+    opacity: 1;
+}
+
+.eq-spectrum-toggle-in-graph svg {
```

Stylelint (17.6.0)

[error] Expected selector ".eq-spectrum-toggle-in-graph svg" to come before selector ".sidebar-nav .nav-item a svg", at line 1176 (no-descending-specificity)

(no-descending-specificity)